internal/runners/branch/list.go

```diff
@@ -0,0 +1,45 @@
+package branch
+
+import (
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/primer"
+	"github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/ActiveState/cli/pkg/project"
+)
+
+type primeable interface {
+	primer.Outputer
+	primer.Projecter
+}
+
+type List struct {
+	out     output.Outputer
+	project *project.Project
+}
+
+func NewList(prime primeable) *List {
+	return &List{
+		out:     prime.Output(),
+		project: prime.Project(),
+	}
+}
+
+func (l *List) Run() error {
+	logging.Debug("ExecuteList")
+
+	project, err := model.FetchProjectByName(l.project.Owner(), l.project.Name())
+	if err != nil {
+		return locale.WrapError(err, "branch_list_project_err", "Could not get details for project: {{.V0}}", l.project.Namespace().String())
+	}
+
+	var branches []string
+	for _, branch := range project.Branches {
+		branches = append(branches, branch.Label)
```

Comment:
Should be able to add highlighting here?

Reply:
Yes, there is no preference in the story here: https://www.pivotaltracker.com/story/show/176470500 as to what colour the highlight should be. I used `ACTIONABLE` but can change it to whatever is more appropriate.

Reply:
Let's use `NOTICE` and also mark it with an asterisk (`*`) in case the user disabled colors.

`ATIONABLE` is more for links, commands, etc. Stuff that's directly actionable.